### PR TITLE
Simplify the Controller lesson

### PR DESCRIPTION
**Pull request by kcleland0818**
This change is being made in the consumer canvas per the conversation on 7/22 to keep the same modules for Blackrock and AWS to streamline.

- Simplify the lesson.
- Create a controller class with only GET operations.
- Explain the `@RestController`, `@GetMapping`, and `@RequestParam` annotations.
- Have the student try to create a method in the controller at the end of the lesson.
- Annotations and content that was removed will be moved or covered in a later lesson.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,227 +3,153 @@
...
-This annotation allows the method to accept JSON formatted client data and
-convert it to a Java object. For example, the JSON object mentioned above would
-be mapped to a `Member` object with the `name` and `email` attributes set to the
-values in the JSON.
+Now that we understand how the controller class works, let's add a couple of more
+methods given the requirements:
 
-## Get Methods
+- Greet the customer.
+- Tell the customer the daily lunch special.
```
Comment (linda-seiter) [inline README.md:89]: Were they supposed to implement the daily lunch special in this lesson?

**Review by linda-seiter — COMMENTED** [review]
Great explanation of the various annotations!

**Review by linda-seiter — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -3,227 +3,156 @@
...
+The `@RestController` combines the following annotations:
 
-```java
-@PostMapping("/members")
-public Member createMember(@RequestBody Member member) {
-    return memberService.createMember(member);
-}
-```
+- `@Controller`: Indicates to Spring that the class represents a controller.
+- `@ResponseBody`: Configures Spring to return a JSON response from the controller
```
Comment (developerjaym) [inline README.md]: This says that it configures Spring to return a JSON response but these examples return a String response.
Could that possibly confuse the students?

File: README.md [hunk 3]
```diff
@@ -3,227 +3,156 @@
...
+  methods instead of view templates, which is the default.
```
Comment (developerjaym) [inline README.md]: Should the students already know what a 'view template' is?

File: README.md (hunk 2 above)
Comment (kcleland0818) [reply README.md]: Removed the word "JSON" from the explanation.

File: README.md (hunk 3 above)
Comment (kcleland0818) [reply README.md]: Removed the "view template" reference and explained how it is bound to the web's response body.

Reference: [ResponseBody Annotation](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/ResponseBody.html)

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]

**Pull request merged**